### PR TITLE
Dev bug

### PR DESCRIPTION
**Pull request by DumouchelP**
* **Please check if the PR fulfills these requirements**
- [ ] The commit messages follows our guidelines (explicit, in english)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] CHANGELOG.md has been updated

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

File: volmdlr/core.py [hunk 1]
```diff
@@ -4403,7 +4418,10 @@ def _extract_shells(self):
     def Volume(self):
         volume=0
         for primitive in self.primitives:
-            volume+=primitive.Volume()
+            try:
+                volume+=primitive.Volume()
+            except:
+                pass
```
Comment (masfaraud) [inline volmdlr/core.py]: Name the exception to handle

File: volmdlr/core.py [hunk 2]
```diff
@@ -753,6 +753,20 @@ class LineSegment2D(Line2D):
...
         Line2D.__init__(self, point1, point2, name = name)
+        
+    def to_dict(self):
+        # improve the object structure ?
+        dict_ = {}
+        dict_['name'] = self.name
+        dict_['point1'] = self.points[0].to_dict()
+        dict_['point2'] = self.points[1].to_dict()
+        dict_['object_class'] = 'volmdlr.core.LineSegment2D'
+        return dict_
```
Comment (masfaraud) [inline volmdlr/core.py]: Why not using generic dict?

File: volmdlr/core.py [hunk 3]
```diff
@@ -4627,7 +4645,7 @@ def babylon_data(self):
                           bbox.zmax - bbox.zmin])
                 
         babylon_data = {'meshes': meshes,
-                        'max_length': max_length,
+                        'max_length': max_length/5.,
```
Comment (masfaraud) [inline volmdlr/core.py]: What is the point of doing this?

**Comment by DumouchelP** [comment]
Update for pull request

**Pull request merged**